packages/home-spa/src/app.js

```diff
@@ -42,8 +39,9 @@ if (footer !== null) {
       </div>
       <div class="footer__links">
         <a href="#">One Portal FAQs</a>
-        <a href="#">File an Issue</a>
-        <a href="mailto:one-portal-devel@redhat.com">one-portal-devel@redhat.com</a>
+        <a href="https://projects.engineering.redhat.com/secure/CreateIssue!default.jspa" rel="noreferrer" target="_blank">File an Issue</a>
```

Comment:
How would the requester know which JIRA project to file the issue for?

Reply:
This is for now. This would later be changed with feedback component. Clicking on this would open the feedback panel and user would be able to add it for the correct JIRA project